### DISCUSSION TITLE
Can I choose to show different plugins controls when lightbox enters full screen mode?

### DISCUSSION THREAD

**Discussion by namannndev** [igordanchenko/yet-another-react-lightbox#311] [Q&A] [answered]
I am trying to implement an inlline lightbox with only Fullscreen option there.
When user clicks on full screen icon & goes to full screen, I want to show other options like download, zoom etc.
How can I do that?

Tried this solution, but facing issue with index selection -

Used a fullscreen state, set it using lightbox event handlers. Showed plugins according to fullscreen state.

```
<Lightbox
.
.
.
on={{ 
enterFullscreen: () =>  setFullScreen(true),
exitFullscreen: () => setFullScreen(false)
}}
plugins={
   isFullScreen
    ? [Inline, Thumbnails, Fullscreen, Zoom, Download]
    : [Inline, Thumbnails, Fullscreen]
}
/> 
```

changing the plugins array with conditions is resetting the selectedSlideIndex to 0.

**Comment by igordanchenko** [comment, marked as answer]
https://yet-another-react-lightbox.com/documentation#Lightbox

Does the description of the `slides` prop answer your question?
